pkg/reconciler/pipelinerun/controller.go

```diff
@@ -99,9 +107,21 @@ func NewController(namespace string, images pipeline.Images) func(context.Contex
 			UpdateFunc: controller.PassNew(impl.EnqueueControllerOf),
 		})
 
-		c.Logger.Info("Setting up ConfigMap receivers")
-		c.configStore = config.NewStore(images, c.Logger.Named("config-store"))
-		c.configStore.WatchConfigs(cmw)
+		go func() {
+			for {
+				select {
+				case <-ctx.Done():
+					// When the context is cancelled, stop reporting.
+					return
+
+				case <-time.After(30 * time.Second):
```

Comment:
Nice :)

We probably need a more general discussion about whether we want to report metrics on events or periodically, and if the latter we should have a central place where we define all the report period, but this should be fine for now.

Reply:
Agreed :+1: I would like for people to configure this period of time, but we can do that in a follow-up

Reply:
I moved this to https://github.com/tektoncd/pipeline/pull/2762